### ISSUE TITLE
Refering to an empy package cpprestsdk

### ISSUE THREAD

**Issue by papaathome** [NuGet/docs.microsoft.com-nuget#2601]
The link to [cpprestsdk](https://www.nuget.org/packages/cpprestsdk/) is stale.
Package revoked by creator and it no longer contains any information.
Refered files mentioned on this page are missing in the package.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c7064d86-7de7-1a58-393b-39a5281a9617
* Version Independent ID: 8efa557b-8b1e-6320-4640-5226b8983832
* Content: [Creating Native NuGet Packages](https://docs.microsoft.com/en-us/nuget/guides/native-packages)
* Content Source: [docs/guides/Native-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/guides/Native-Packages.md)
* Product: **nuget**
* GitHub Login: @JonDouglas
* Microsoft Alias: **jodou**

**Comment by joelverhagen** [comment]
Yes, it appears that the package has been unlisted by the owner on NuGet.org. @JonDouglas, @clairernovotny (author and recent editor) -- do we know if there is an alternative to this package? No deprecation status is set on the package so it's hard for users to find an alternative.

**Comment by clairernovotny** [comment]
The current page points to [WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2), is this issue still unresolved?

https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/guides/Native-Packages.md

**Comment by papaathome** [comment]
Looks like it it workable now. It needs any visitor to pay attention to the details and look in the new versions tab.
